### ISSUE TITLE
interactive search should clear search field on new search

### ISSUE THREAD

**Issue by robmcmullen** [robmcmullen/peppy#581]
_[Trac time 20090109 213822Z]_ From Chris Barker on the peppy mailing list:
- hit cmd-F to start searching
- type in my search string
- when I've found what I want, click on the main editing window, and edit away.
- hit cmd-F to search for something else
- type in my search string -- WHOOPS! the old one is still there, so I now need to go back and delete that, and start searching again.

What I'd like is when I hit the keystroke to start the search, and the focus shifts to the existing search field , it would get cleared. I could always hit the search key-stroke again to get the last search back if I wanted that.

In short, it should behave the same way, whether or not the search
mini-window was already there or or not.

**Comment by robmcmullen** [comment]
_[Trac time 20090605 172246Z]_ Firefox style works on Mac, too.

**Comment by robmcmullen** [comment]
_[Trac time 20090109 220753Z]_ GTK has firefox-style behavior where the text in the search field get selected when the focus changes to it; windows and Mac don't.  If windows and Mac could be made to emulate this behavior, it would be sufficient.

**Comment by robmcmullen** [comment]
_[Trac time 20090212 050354Z]_ 2e7fa2935966f712f9560de52bc500e7d9b90d47 Refs #581: implemented firefox style behaviour for interactive search box on win
